src/Controller/ColorController.php

```diff
@@ -33,16 +33,23 @@ public function create(
             $colorDTO = $serializer->deserialize($request->getContent(), ColorDTO::class, 'json');
 
             $errors = $validator->validate($colorDTO);
-
             if ($errors->count()) {
-                return $this->json($errors[0]->getMessage(), Response::HTTP_CONFLICT);
+                return $this->json(
+                    [
+                    'status' => Response::HTTP_CONFLICT,
+                    'message' => $errors[0]->getMessage()
+                    ],
+                    Response::HTTP_CONFLICT
```

Comment:
Tu n'as pas besoin de mettre la key `status` une fois que tu ajoute 2em paramètre de `$this->json()`. 
A la place de `status => Resonse::HTT*` tu mets `success => false`